test/unit/model/v2/test_platforms_section.py

```diff
@@ -175,6 +177,9 @@ def _model_platforms_docker_errors_section_data():
             'volumes': [
                 int(),
             ],
+            'keep_volumes': [
+                bool(),
```

Comment:
Should this not be another type like `int()`?

Reply:
I am a little confused why the tests passed?

Reply:
Same here. This should not result in a 'must be of boolean type' error, I'd expect validation to succeed here based on the schema modifications.

Reply:
Fixed.